app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -125,6 +125,10 @@ def billed?
           !!doc_auth_result
         end
 
+        def doc_type_supported?
+          !doc_class_name.present? || ID_TYPE_SLUGS.key?(doc_class_name) || doc_class_name == 'Unknown'
```

Comment:
Is `unknown` what's returned if it's a passport or a type of ID that's not a state ID?

Reply:
@night-jellyfish , if it's a passport or something else, it should be identified if the document image quality is good.

We can only really say it's not supported if it's clearly identified as an unsupported type.